src/js/Players/ComputerPlayer.js

```diff
@@ -0,0 +1,52 @@
+import {Player} from './Player.js';
+// import {random} from '../../../node_modules/lodash/random.js';
+
+const _ = window._;
+
+export class ComputerPlayer extends Player{
+    constructor(name, sign){
+        super(name, sign);
+    }
+
+    /**
+     * Using currying, Generating a random cell id
+     * @param {number} maxNum number of rows/columns of board
+     * @returns {function} function who generates a random cell id that contains 4 digits
+     */
+    curriedGenerateCellId(maxNum){
+        return ()=>{
+            let cellId = "";
+
+            // bulding cell id  (i < 4 for: 4 digits of cell id (outerRow, outerCol, innerRow, innerCol))
+            for (let i = 0; i < 4; i++){
+                // Returns a random number between min (included) and max (excluded):
+                cellId += _.random(maxNum - 1);
+
+            }
+            return cellId;
+        };
+    }
+
+    /**
+     *
+     * @param {number} maxNum number of rows/columns of board
+     * @returns {number} cell id
+     */
+    async selectCellAsync(maxNum){
```

Comment:
It would the computer player ignores which cells are not available when selecting one randomly - this means you can have infinite randomizer loops, and in practice the loops will take longer the more the game goes on